bin/update-plugins.py

```
@@ -0,0 +1,31 @@
+
+parser = argparse.ArgumentParser(
+    prog="update-plugins",
+    description="Update all plugins in the flake")
+
+
+LOCAL_PINS = "npins/sources.json"
+PINS = os.path.join(os.getcwd(), local_pins)
+
+
+def main():
+    args = parser.parse_args()
```

Ruff (0.8.2)

Local variable `args` is assigned to but never used

Remove assignment to unused variable `args`

(F841)